cmd/helm/downloader/manager.go

```diff
@@ -295,6 +311,22 @@ func (m *Manager) getRepoNames(deps []*chartutil.Dependency) (map[string]string,
 	// by Helm.
 	missing := []string{}
 	for _, dd := range deps {
+		// if dep chart is from local path, verify the path is valid
+		if strings.HasPrefix(dd.Repository, "file://") {
+			depPath, err := filepath.Abs(dd.Repository[7:])
+			if err != nil {
+				return nil, err
+			}
+
+			if _, err = os.Stat(depPath); os.IsNotExist(err) {
+				return nil, fmt.Errorf("directory %s not found: %s", depPath, err)
```

Comment:
This error should probably be a little bit more human friendly. As of now it looks like: `Error: directory /Users/test/go/src/k8s.io/helm/docs/examples/blahalpine not found: stat /Users/test/go/src/k8s.io/helm/docs/examples/blahalpine: no such file or directory`.

I would recommend just not printing the error returned from `os.Stat`